SUIApp/SUIApp/Views/VerificationView.swift

```diff
@@ -0,0 +1,177 @@
+// VerificationView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран верификации
+struct VerificationView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let oneText = "1"
+        static let threeText = "3"
+        static let sixText = "6"
+        static let nineText = "1"
+        static let zeroText = "0"
+        static let okText = "Ок"
+        static let emptyText = ""
+        static let fillFromMessageText = "Fill in from message?"
+        static let sendSmsText = "Send sms again"
+        static let notReciveSmsText = "Did’t receive sms?"
+        static let actionSheetTitleText = "We already know about the bug and fix it sooon"
+        static let continueText = "Continue"
+        static let checkSmsText = "Check the SMS"
+        static let descriptionText = "massageto get verification code"
+        static let verificationCodeText = "Verification Code"
+        static let navigationTitleText = "Verification"
+    }
+
+    private enum States {
+        case one, two, three, four
+    }
+
+    var body: some View {
+        ZStack {
+            VStack(spacing: 30) {
+                Rectangle()
+                    .fill(Color.clear)
+                    .frame(height: 10)
+                    .background(gradient)
+                Image(systemName: verificationViewModel.fetchImage(flag: isAlertShown))
+                    .resizable()
+                    .frame(width: 150, height: 100)
+                verificationCodeView
+                checkSmsView
+                sendSmsView
+                Spacer()
+            }
+            .navigationTitle(Constants.navigationTitleText)
+        }
+    }
+
+    @StateObject private var verificationViewModel = VerificationViewModel()
+    @State private var oneCode = Constants.emptyText
+    @State private var twoCode = Constants.emptyText
+    @State private var threeCode = Constants.emptyText
+    @State private var fourCode = Constants.emptyText
+    @State private var isAlertShown = false
+    @State private var isActionSheetShown = false
+    @FocusState private var focused: States?
```

Comment:
```suggestion
    @StateObject private var verificationViewModel = VerificationViewModel()
 
    @FocusState private var focused: States?
```
Это тоже похоже на модель